docs/learn-DOT.md

```diff
@@ -49,6 +53,59 @@ DOTs will be used to facilitate the consensus mechanism that underpins Polkadot.
 
 DOTs will have the ability to be bonded for a duration of time in order to add a new parachain to the network. The DOTs will be locked during their bonding period and will be released back to the account that bonded them after the duration of the bond has elapsed and the parachain is removed.
 
+### Vesting
+
+DOTs that are allocated to the Polkadot team or awarded to teams as part of 
+Web3 Foundation's grants program will have a vesting schedule attached. Vested
+balances are classified as a separate abstraction to the other locks that are
+placed on your balance. This means that although vested tokens cannot be transferred,
+they can still be used for voting in governance or staked in the staking system.
+
+Vesting schedules are linear release and will unlock a constant amount of tokens
+at every block until all the funds have been made available.
+
+Let's take for example the data structure for the vesting schedule, the `VestingInfo`
+struct:
+
+```rust
+pub struct VestingInfo<Balance, BlockNumber> {
+	/// Locked amount at genesis.
+	pub locked: Balance,
+	/// Amount that gets unlocked every block after `starting_block`.
+	pub per_block: Balance,
+	/// Starting block for unlocking(vesting).
+	pub starting_block: BlockNumber,
+}
+```
```

Comment:
I'm opposed to showing code like this. It's out of context and does nothing to make the idea more concrete. You can easily explain this with natural language. Vesting is a simple concept and any user, no matter how technical, should be able to understand it from this section.